src/main/java/org/spongepowered/api/item/inventory/type/Inventory2D.java

```diff
@@ -44,7 +44,9 @@
      * @param pos Slot position to query
      * @return matching stacks, as per the semantics of {@link Inventory#poll()}
      */
-    Optional<ItemStack> poll(SlotPos pos);
+    default Optional<ItemStack> poll(Vector2i pos) {
```

Comment:
Likewise here, changes that aren't quite relevant to the PR.

Reply:
I changed `SlotPos` to `Vector2i`, but I can get rid of the default impl if you want